gradio/blocks.py

```diff
@@ -207,9 +214,11 @@ def get_config(self):
             to_add = e.config_data()
             if to_add:
                 config = {**to_add, **config}
-        config.pop("_skip_init_processing", None)
         config.pop("render", None)
-        return {**config, "root_url": self.root_url, "name": self.get_block_name()}
+        config = {**config, "proxy_url": self.proxy_url, "name": self.get_block_name()}
+        if (_selectable := getattr(self, "_selectable", None)) is not None:
```

Comment:
The := is not valid until 3.10 I think.

I think this looks good, I don't like treating some props as special but I guess we can make an exception here since they're pretty opaque parameters.

I think we need to change "selectable" to "_selectable" in the event callback so that it gets set

```python
    select = EventListener(
        "select",
        callback=lambda block: setattr(block, "_selectable", True),
        doc="Event listener for when the user selects or deselects the {{ component }}. Uses event data gradio.SelectData to carry `value` referring to the label of the {{ component }}, and `selected` to refer to state of the {{ component }}. See EventData documentation on how to use this event data",
    )
```

Reply:
https://www.geeksforgeeks.org/walrus-operator-in-python-3-8/ looks like Python 3.8 should support walrus

Reply:
Ah interesting, I thought it was set but I'll confirm

Reply:
> looks like Python 3.8 should support walrus

nice

Reply:
Done